app/js/controllers.js

```diff
@@ -8,6 +9,20 @@ conceptDictControllers.controller('ClassesListCtrl', ['$scope', 'loadClasses', '
 		$location.path( hash );
 	};
 	
+	//holds objects of selected checkboxes
+	$scope.selected = {};
+	
+	//deletes selected classes
+	$scope.deleteSelected = function(){
+	    angular.forEach($scope.selected, function(key,value){
+	        if(key){
+	        	ClassesService.deleteClass({uuid : value});
+	        }
+	            
+	    });
+	    //reloads the page ($route.reload doesn't get updated list)
+	    location.reload();
```

Comment:
Is the whole page reloaded? If yes, then maybe it's better to just do ClassesService.getAll().then(function(data) {
$scope.classes = data });